engine-transactions/src/legacy.rs

```diff
@@ -45,20 +46,20 @@ impl TransactionLegacy {
         self.gas_limit.try_into().ok()
     }
 
-    pub fn normalize(self) -> super::NormalizedEthTransaction {
-        super::NormalizedEthTransaction {
-            address: None,
-            chain_id: None,
-            nonce: self.nonce,
-            gas_limit: self.gas_limit,
-            max_priority_fee_per_gas: self.gas_price,
-            max_fee_per_gas: self.gas_price,
-            to: self.to,
-            value: self.value,
-            data: self.data,
-            access_list: Vec::new(),
-        }
-    }
+    // pub fn normalize(self) -> super::NormalizedEthTransaction {
```

Comment:
Please delete commented out code if it is not needed.

Reply:
Its still in draft, got it like this for now until I fix these tests.